components/edit-collective/EditPaymentMethod.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { matchPm, PAYMENT_METHOD_SERVICE, PAYMENT_METHOD_TYPE } from '../../lib/constants/payment-methods';
+import { paymentMethodLabelWithIcon } from '../../lib/payment_method_label';
+
+import { Box, Flex } from '../Grid';
+import Link from '../Link';
+import StyledButton from '../StyledButton';
+
+type EditPaymentMethodProps = {
+  collectiveSlug: string;
+  isSaving: boolean;
+  needsConfirmation: boolean;
+  nbActiveSubscriptions: number;
+  paymentMethod: any; // TODO Replace with PaymentMethod type once parent will be typed
```

Comment:
Can't we fetch this from the types being automatically generated?

Reply:
Absolutely, I was thinking that we needed for the query in `components/edit-collective/sections/PaymentMethods.js` to be typed but we already have what we need in `components/edit-collective/sections/PaymentMethods.js`. Pushing an update.